### PR TITLE
Augment tracing configurability doc

### PR DESCRIPTION
**Pull request by nrjpoddar**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Fixes https://github.com/istio/istio.io/issues/7902

### CODE REVIEW

**Comment by nrjpoddar** [comment]
cc @jacob-delgado @ericvn

**Comment by nrjpoddar** [comment]
@douglas-reid or @mandarjog can you review this? Thanks!

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md [hunk 1]
```diff
@@ -8,13 +8,49 @@ test: no
...
+    based on your preferred tracing backend to install the appropriate addon and
+    configure your Istio proxies to send traces to the tracing deployment.
+
+1.  Ensure that your applications propagate tracing headers as described [here](/docs/tasks/observability/distributed-tracing/overview/).
+
+## Available tracing configurations
+
+You can configure the following tracing options in Istio:
+
+1.  Random sampling rate for percentage of requests that will selected for trace
```
Comment (frankbu) [inline content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:27]:
```suggestion
1.  Random sampling rate for percentage of requests that will be selected for trace
```

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md [hunk 2]
```diff
@@ -8,13 +8,49 @@ test: no
...
+{{< warning >}}
+In order for the new tracing configuration to take effect for either of these
+options you need to restart pods injected with Istio proxies.
+{{< /warning >}}
+
+### Using `MeshConfig` for trace settings
+
+All tracing options can be configured globally via `MeshConfig`.
+To simplify configuration, it is recommended to create a single YAML file
+which you can pass to `istioctl install -f` command.
```
Comment (frankbu) [inline content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:53]:
```suggestion
which you can pass to the `istioctl install -f` command.
```

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md [hunk 3]
```diff
@@ -50,34 +87,34 @@ spec:
...
-Istio captures a trace for all requests by default when installing with the demo profile.
-For example, when using the Bookinfo sample application, every time you access
-`/productpage` you see a corresponding trace in the
-[dashboard](../jaeger/). This sampling rate is suitable for a test or low traffic
-mesh. For a high traffic mesh you can lower the trace sampling
-percentage in one of two ways:
+The sampling rate option can be used to control what percentage of requests get
+reported to your tracing system. This should be configured depending upon your
+traffic in the mesh and the amount of tracing data you want to collect.
+The default rate is 1%:
```
Comment (frankbu) [inline content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:104]:
```suggestion
The default rate is 1%.
```

**Comment by nrjpoddar** [comment]
@frankbu thanks for your review! Can I get an approval?

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md [hunk 4]
```diff
@@ -8,13 +8,49 @@ test: no
...
-## Create a `MeshConfig` with trace settings
+## Before you begin
 
-All tracing options are configured by using `MeshConfig` during Istio *installation*.
-To simplify configuration, create a single YAML file to pass to `istioctl`.
+1.  Follow the tracing installation guide located under [Integrations](/docs/ops/integrations/)
+    based on your preferred tracing backend to install the appropriate addon and
+    configure your Istio proxies to send traces to the tracing deployment.
+
+1.  Ensure that your applications propagate tracing headers as described [here](/docs/tasks/observability/distributed-tracing/overview/).
```
Comment (douglas-reid) [inline content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:21]: i wonder if we should switch the ordering here. Updating apps to propagate context is probably the first thing that should be done.

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md [hunk 5]
```diff
@@ -24,19 +60,20 @@ spec:
   meshConfig:
     defaultConfig:
       tracing:
+        sampling: 10
+        custom_tags:
+          my_header:                    # user-defined name
```
Comment (douglas-reid) [inline content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:65]: nit: worth saying `# user-defined tag name` ?

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md (hunk 4 above)
Comment (nrjpoddar) [reply content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:21]: make sense, will switch it.

File: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md (hunk 5 above)
Comment (nrjpoddar) [reply content/en/docs/tasks/observability/distributed-tracing/configurability/index.md:65]: Agree

**Pull request merged**